README.md

```diff
@@ -15,19 +15,18 @@
 
 ## Installation / Setup of this app
 
-This set of installation steps assume that you already have set up Nextcloud using a web server like Apache or Nginx and will help you to set up Spreed WebRTC below the `/webrtc` subpath at your existing web server. If your Nextcloud server is not using SSL yet, you need to [enable SSL now](https://docs.nextcloud.com/server/9/admin_manual/installation/source_installation.html#enabling-ssl-label). Spreed WebRTC does require an encrypted connection and
-as this app is running within Nextcloud that means that your Nextcloud needs to
-be accessed with HTTPS.
+This set of installation steps assume that you already have set up Nextcloud using a web server like Nginx or Apache.  
+Your server has to be available via HTTPS. If your Nextcloud server is not using SSL/TLS yet, you need to [enable SSL now](https://docs.nextcloud.com/server/9/admin_manual/installation/source_installation.html#enabling-ssl-label).
 
-1. Place this app in the **apps/** folder of your Nextcloud installation.
+1. Place this app in the **apps/** folder of your Nextcloud installation. Note that this app's directory has to be named `spreedme` (this only should affect you if you cloned it from Github).
```

Comment:
Please just tell the user what to do. Like "Make sure the directory is named `spreedme`.